extensions/coze/src/services/config.tsx

```diff
@@ -0,0 +1,22 @@
+import { getPreferenceValues } from "@raycast/api";
+
+interface Preferences {
+  api_base: string;
+  debug: boolean;
+}
+
```

Comment:
You don't need to manually set preference types as this is autogenerated in `raycast-env.d.ts`x

Reply:
i remove type, but it error:

```
Error: src/services/config.tsx(4,3): error TS7008: Member 'api_base' implicitly has an 'any' type.
Error: src/services/config.tsx(5,3): error TS7008: Member 'debug' implicitly has an 'any' type.
```

Reply:
Sounds like you need to restart he TS server

<img width="651" alt="image" src="https://github.com/user-attachments/assets/3dbc0705-d2d3-4459-9962-47233a1f4d00">

And then you can do it like this (also note that 'await' doesn't do anything)

<img width="674" alt="image" src="https://github.com/user-attachments/assets/e316e343-c7cb-492e-9db3-3a23dda338be">

Reply:
thanks, i fix it.